src/parser/jobs/nin/modules/Ninki.js

```diff
@@ -104,4 +121,26 @@ export default class Ninki extends Module {
 			</Trans>,
 		}))
 	}
+
+	output() {
+		const _ninkiColor = Color(JOBS.NINJA.colour)
```

Comment:
Super minor, but this should just be `ninkiColor`, no `_`. The underscore is convention for private instance variables, not local ones.

Reply:
All above fixed in 95923f4.